.github/workflows/e2e.yml

```
@@ -110,9 +112,22 @@ jobs:
     secrets:
       DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
       DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+  
+  e2e-s3-vcluster:
+    if: ${{ ! github.event.pull_request.head.repo.fork  && (inputs.e2e_test_suites == 'vcluster s3' || inputs.e2e_test_suites == 'leg 2 vcluster' || inputs.e2e_test_suites == '')}}
```

Why do we check for two test suites?
```suggestion
    if: ${{ ! github.event.pull_request.head.repo.fork  && (inputs.e2e_test_suites == 'vcluster s3' || inputs.e2e_test_suites == '')}}
```